pkgs/build-support/rust/build-rust-crate/default.nix

```diff
@@ -329,7 +329,7 @@ crate_: lib.makeOverridable
       colors = lib.attrByPath [ "colors" ] "always" crate;
       extraLinkFlags = lib.concatStringsSep " " (crate.extraLinkFlags or [ ]);
       edition = crate.edition or null;
-      codegenUnits = if crate ? codegenUnits then crate.codegenUnits else 1;
+      codegenUnits = crate.codegenUnits or (1);
```

Comment:
```suggestion
      codegenUnits = crate.codegenUnits or 1;
```

Reply:
I am not sure why the linter suggested that to me.